src/app/itemsOnLifecycle.clj

```diff
@@ -190,3 +192,42 @@
    classes-using-finalizers
    []
    [:class]))
+
+; ============================================
+; ITEM 10
+; ============================================
+
+(defn- overrides-toString? [class]
+  (->> (getMethods class)
+       (filter #(= (getName %) "toString"))
```

Comment:
does it make sense to verify if the method is abstract?
I think it will be pretty common to have such cases in which we want to find methods considering name and parameters. We should probably adapt the function `find-methods-by-signature` in find.clj to work not only on cu but also on classes and use it in this case.